ftpretty.py

```diff
@@ -31,18 +31,25 @@
 class ftpretty(object):
     """ A wrapper for FTP connections """
     conn = None
+    port = 21
     tmp_output = None
     relative_paths = set(['.', '..'])
 
     def __init__(self, host, user, password,
             secure=False, passive=True, ftp_conn=None, **kwargs):
 
+        if 'port' in kwargs.keys():
+            self.port = kwargs['port']
+            del kwargs['port']
+
         if ftp_conn:
             self.conn = ftp_conn
         elif secure and FTP_TLS:
+            FTP_TLS.port = self.port
```

Comment:
if self.port is not None:

Reply:
Can we just have `if self.port:` to also exclude falsy values?